mappings/net/minecraft/client/render/GameRenderer.mapping

```diff
@@ -308,7 +323,14 @@ CLASS net/minecraft/unmapped/C_sktkdoun net/minecraft/client/render/GameRenderer
 	METHOD m_vtnchlsw (Lnet/minecraft/unmapped/C_alllhitb;)V
 		ARG 0 shader
 	METHOD m_vwvkdyvv getPositionColorLightmapShader ()Lnet/minecraft/unmapped/C_alllhitb;
+	METHOD m_vzamypqw getPotentialHit (Lnet/minecraft/unmapped/C_astfners;DDF)Lnet/minecraft/unmapped/C_eetoegzn;
+		ARG 1 entity
+		ARG 2 blockRange
+		ARG 4 entityRange
+		ARG 6 tickDelta
 	METHOD m_vzhzwpiw getRenderTypeEntityAlphaShader ()Lnet/minecraft/unmapped/C_alllhitb;
+	METHOD m_vziixqvb (Lnet/minecraft/unmapped/C_astfners;)Z
+		ARG 0 e
```

Comment:
```suggestion
		ARG 0 entity2
```
would be more in line with other lambda params I've seen in QMap, but we should probably clarify what to do for these in a convention.